back/spec/policies/idea_policy_spec.rb

```diff
@@ -14,29 +14,29 @@
     context 'for a visitor' do
       let(:user) { nil }
 
-      it { is_expected.to permit(:show)           }
-      it { is_expected.to permit(:by_slug)        }
-      it { is_expected.not_to permit(:create)     }
-      it { is_expected.not_to permit(:update)     }
-      it { is_expected.not_to permit(:destroy)    }
-      it { is_expected.not_to permit(:index_xlsx) }
-
-      it 'indexes the idea' do
+      it do
```

Comment:
It sped up this spec file from ~20 to ~4 seconds. Also, added it to BE CDM https://www.notion.so/govocal/Back-end-Code-Discussion-Meeting-BE-CDM-a94ef3cfdd9849e4bde22d2b10dcffeb?pvs=4#f40144f09bcd464aa7c34ec9ff754716